### PR TITLE
Bug Fix: Remove invalid comma

### PR DESCRIPTION
**Pull request by erikerikson**
Remove invalid comma (line 83)
Fix eslint.  How'd that happen?  /shrug
It'll be nice to get the bikeshed PR merged.

### CODE REVIEW

File: product-catalog/catalog.js [hunk 1]
```diff
@@ -148,20 +148,24 @@ const impl = {
...
+        complete(`${constants.METHOD_PROCESS_EVENT} ${constants.BAD_MSG
+        } could not validate event to '${productCreateSchema}' schema. Errors: ${ajv.errorsText()}`);
       } else {
         impl.putProduct(event, complete);
       }
     } else {
       // TODO remove console.log and pass the above message once we are only receiving subscribed events
-      console.log(`${constants.MODULE} ${constants.METHOD_PROCESS_EVENT}  ${constants.BAD_MSG}- event with unsupported schema (${event.data.schema}) observed.`);
+      console.log(`${constants.MODULE} ${constants.METHOD_PROCESS_EVENT} ${constants.BAD_MSG
+      } - event with unsupported schema (${event.data.schema}) observed.`);
```
Comment (gwsii) [inline product-catalog/catalog.js]: Ugh. Not liking how these lines were wrapped at all. You floated the idea of no line limits, and I think that's fine.

Comment (erikerikson) [inline product-catalog/catalog.js]: yeah, once the bikeshedding PR is merged...  with a deadline of Thursday (2017/01/26)

Comment (erikerikson) [inline product-catalog/catalog.js]: then I'll just make them single lines. 'cause yeah.  it's fugly and silly

Comment (gwsii) [inline product-catalog/catalog.js]: Ship it!

**Pull request merged**